controllers/databaseclusterbackup_controller.go

```diff
@@ -153,25 +161,223 @@ func (r *DatabaseClusterBackupReconciler) SetupWithManager(mgr ctrl.Manager) err
 	if err == nil {
 		if err := r.addPXCToScheme(r.Scheme); err == nil {
 			controller.Owns(&pxcv1.PerconaXtraDBClusterBackup{})
+			controller.Watches(
+				&pxcv1.PerconaXtraDBClusterBackup{},
+				handler.EnqueueRequestsFromMapFunc(func(ctx context.Context, obj client.Object) []reconcile.Request {
+					return r.tryCreateDBBackups(ctx, obj, r.tryCreatePXC)
+				}))
 		}
 	}
 
 	err = r.Get(ctx, types.NamespacedName{Name: psmdbBackupCRDName}, unstructuredResource)
 	if err == nil {
 		if err := r.addPSMDBToScheme(r.Scheme); err == nil {
 			controller.Owns(&psmdbv1.PerconaServerMongoDBBackup{})
+			controller.Watches(
+				&psmdbv1.PerconaServerMongoDBBackup{},
+				handler.EnqueueRequestsFromMapFunc(func(ctx context.Context, obj client.Object) []reconcile.Request {
+					return r.tryCreateDBBackups(ctx, obj, r.tryCreatePSMDB)
+				}))
 		}
 	}
 
 	err = r.Get(ctx, types.NamespacedName{Name: pgBackupCRDName}, unstructuredResource)
```

Comment:
The watcher is added only if the `addPSMDBToScheme()` didn't throw any error which it would if there is no CRD available.

If the CRD installed after we start watching - this is broader problem which is relevant for the DBclusters too. I remember we had a conversation with Diogo about this and he was going to create a ticket for it. @recharte did you have a chance to create it?

Reply:
Would the watch fail if the CRD was not in place yet? If not, we could initiate the watch regardless if CRD has been already installed, right?

Reply:
we can not initiate the watch for a resource which is not available in the scheme, it will cause an error. 

```
kind must be registered to the Scheme {"error": "no kind is registered for the type v1.PerconaServerMongoDBBackup in scheme \"pkg/runtime/scheme.go:100\""}
```